lib/beacon/content.ex

```diff
@@ -1457,6 +1457,14 @@ defmodule Beacon.Content do
     |> Component.changeset(attrs)
     |> validate_component_body()
     |> Repo.update()
+    |> case do
+      {:ok, component} ->
+        :ok = PubSub.component_updated(component)
+        {:ok, component}
+
+      error ->
+        error
+    end
   end
```

Comment:
```suggestion
    |> tap(&maybe_reload_component/1)
  end
  
  def maybe_update_component({:ok, component}), do: PubSub.component_updated(component)
  def maybe_update_component({:error, _component}), do: :noop
```
This might be controversial, but WDYT about `tap/2` to make it clear that the result of `Repo.update/2` will not be changed?  I think I slightly prefer it, but either way is fine, so feel free to disregard this if you don't agree 🙂

Reply:
That's not a pattern I've seen used in other projects too much but I'm willing to give it a try